CHANGES.md

```diff
@@ -2,6 +2,11 @@
 Change Log
 ==========
 
+### 2018-07-16-updated-2018-10-03
```

Comment:
It'll be 2018-09-14-updated-2018-10-03, even though I didn't add the 2018-09-14 catalog to changelog (because there were no changes, there was a new version so the new code wouldn't share a version with the old code. I'm not sure if I'm doing this right.)

Reply:
Fixed in new commit.